test5.cpp

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+
+int
+main()
+{
+    int a = 0;
+    printf("hello,world!\n");
+
+    a = 1;
+
+    printf("testtest %d\n",a);
+
+    return 0;
+
+
+
+    
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]<br>  Blank line at the end of a code block.  Is this needed?  [whitespace/blank_line] [3]